src/datasets/loaders/openproblems_neurips2021_bmmc/script.py

```diff
@@ -30,10 +31,20 @@ def remove_mod_prefix(df, mod):
   suffix = f"{mod}_"
   df.columns = df.columns.str.removeprefix(suffix)
 
+def convert_matrix(adata):
+  for key in adata:
+      if isinstance(adata[key], sparse.csr_matrix):
+        adata[key] = sparse.csc_matrix(adata[key])
+  return adata
+      
 
 print("load dataset file", flush=True)
 adata = ad.read_h5ad(par["input"])
 
+# Convert to sparsce csc_matrix
+adata.layers = convert_matrix(adata.layers)
+adata.obsm = convert_matrix(adata.obsm)
+
```

Comment:
inplace is also fine

```suggestion
convert_matrix(adata.layers)
convert_matrix(adata.obsm)

```